src/index.js

```diff
@@ -333,20 +312,16 @@ function ncc (
             loader: eval('__dirname + "/loaders/notfound-loader.js"')
           }]
         },
+        {
```

Comment:
use node-loader instead of relocate-loader for simple case, reloacate-loader is complex and no needed for most nodejs application, but we will consider compatible with it in the future